### ISSUE TITLE
[Accessibility]: H1 focus outline cut off when using skip link in form flow

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/vets-design-system-documentation#5221]
### Point of contact

Jamie Klenetsky Fay

### How severe do you think this issue is?

Medium

### Severity rationale

Missing focus outlines are "critical" (defect level 1), but _most_ of the outline is present here, it's just cut off.

### URL where issue occurs

https://staging.va.gov/family-and-caregiver-benefits/survivor-compensation/apply-for-accrued-benefits-form-21p-601/personal-information

### Component or pattern (if applicable)

I believe this involves va-breadcrumb in some way, but not sure.

### Environment details

- OS: Mac OS 26.1
- Browser: Chrome

### Steps to reproduce

1. Shift-tab until you can access the "Skip to content" link
2. Press Enter to select the link

### Observed behavior

Focus correctly moves to the H1, but the outline itself is cut off. It's only present on 3 sides - the top of the outline appears to be cut off, maybe covered up by the breadcrumb component.

### Expected behavior / Acceptance criteria

The H1's focus outline is present on all 4 sides.

### Supporting evidence (screenshots, recordings, code snippets)

<img width="611" height="227" alt="Image" src="https://github.com/user-attachments/assets/d47f3755-ed04-4498-892c-14a5a5041bf5" />

### Type of issue

- [ ] Axe-core
- [ ] Cognitive | user flows & navigating
- [x] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Content style guide
- [x] Focus
- [x] Headings
- [x] Keyboard
- [ ] Link and button labels
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Screenreader | assistive tech & device support
- [ ] Semantic HTML
- [ ] URL standards
- [ ] Zoom or Magnification
- [ ] Something else
